packages/atlas/src/components/_comments/Comment/Comment.tsx

```diff
@@ -44,7 +56,7 @@ export const Comment: React.FC<CommentProps> = React.memo(
 
     const { activeMemberId, activeMembership, activeAccountId, signIn } = useUser()
     const { comment } = useComment(
-      { commentId: commentId ?? '', memberId: activeMemberId ?? undefined, videoId: video?.id },
+      { commentId: commentId ?? '' },
       {
         skip: !commentId,
         fetchPolicy: 'cache-only',
```

Comment:
actually, do we still need this fetch policy? I commented on this line and I didn't notice much difference in terms of the number of requests.

Reply:
There shouldn't be a difference if everything works correctly. The difference would be in the case when data wasn't properly fetched outside of the component. But I'll get rid of it